### PR TITLE
Improve README

### PR DESCRIPTION
**Pull request by delvedor**
Now the README is more user-friendly, it highlights better where to find the full documentation and offers a complete quick start example.

### CODE REVIEW

**Comment by mcapodici** [comment]
@delvedor - this looks great. Thank you for making these changes.

File: README.md [hunk 1]
```diff
@@ -29,62 +30,49 @@ npm install @elastic/elasticsearch
...
-const { Client: Client7 } = require('es7')
 
-const client6 = new Client6({ node: 'http://localhost:9200' })
-const client7 = new Client7({ node: 'http://localhost:9201' })
+#### Browser
 
-client6.info(console.log)
-client7.info(console.log)
-```
+There is no official support for the browser environment since expose your Elasticsearch instance to everyone could lead to security issues.
```
Comment (philkra) [inline README.md]: > **Warning**: There is no official support for the browser environment since expose your Elasticsearch instance to everyone could lead to security issues.

File: README.md [hunk 2]
```diff
@@ -1,23 +1,24 @@
...
 
+The official Node.js client for Elasticsearch.
+
 ---
 
-**Note:** In the past months we have worked on the new Elasticsearch Node.js client, and if you want you can already try it by following the instructions below, while if you're going to use the legacy one or report an issue, please check out [elastic/elasticsearch-js-legacy](https://github.com/elastic/elasticsearch-js-legacy).
+**Note:** In the past months we have worked on the new Elasticsearch Node.js client, and can use it by following the instructions below, while if you're going to use the legacy one or report an issue, please check out [elastic/elasticsearch-js-legacy](https://github.com/elastic/elasticsearch-js-legacy).
```
Comment (lcawl) [inline README.md]: Drive-by suggestion:
```suggestion
**Note:** In the past months we have worked on the new Elasticsearch Node.js client and you can use it by following the instructions below. If you're going to use the legacy one or report an issue, however, please check out [elastic/elasticsearch-js-legacy](https://github.com/elastic/elasticsearch-js-legacy).
```

File: README.md (hunk 1 above)
Comment (lcawl) [inline README.md]:
```suggestion
There is no official support for the browser environment. It exposes your Elasticsearch instance to everyone, which could lead to security issues.
```

File: README.md [hunk 3]
```diff
@@ -29,62 +30,49 @@ npm install @elastic/elasticsearch
...
 
-const client6 = new Client6({ node: 'http://localhost:9200' })
-const client7 = new Client7({ node: 'http://localhost:9201' })
+#### Browser
 
-client6.info(console.log)
-client7.info(console.log)
-```
+There is no official support for the browser environment since expose your Elasticsearch instance to everyone could lead to security issues.
+We recommend you to write a lightweight proxy that uses this client instead.
```
Comment (lcawl) [inline README.md]:
```suggestion
We recommend that you write a lightweight proxy that uses this client instead.
```

File: README.md [hunk 4]
```diff
@@ -29,62 +30,49 @@ npm install @elastic/elasticsearch
...
-You can find the full documentation in our [docs](https://www.elastic.co/guide/en/elasticsearch/client/javascript-api/current/index.html) website.
+## Quick start
 
+First of all, require the client and initialize it:
 ```js
 const { Client } = require('@elastic/elasticsearch')
 const client = new Client({ node: 'http://localhost:9200' })
+```
 
+You can use both the callback-style API and the promise-style API, both behaves in the same way.
```
Comment (lcawl) [inline README.md]:
```suggestion
You can use both the callback-style API and the promise-style API, both behave the same way.
```

File: README.md [hunk 5]
```diff
@@ -109,85 +97,99 @@ The returned value of **every** API call is formed as follows:
...
+npm install es7@npm:@elastic/elasticsearch@7
+```
+And your `package.json` will look like the following:
+```json
+"dependencies": {
+  "es6": "npm:@elastic/elasticsearch@^6.7.0",
+  "es7": "npm:@elastic/elasticsearch@^7.0.0"
 }
 ```
+And finally, you will require the packages from your code by using the alias you have defined.
```
Comment (lcawl) [inline README.md]: Omitting "finally" since the next paragraph uses it too and there can only be one final step
```suggestion
You will require the packages from your code by using the alias you have defined.
```

File: docs/introduction.asciidoc [hunk 6]
```diff
@@ -18,18 +19,129 @@ npm install @elastic/elasticsearch
...
+npm install @elastic/elasticsearch@<major>
 ----
-# Elasticsearch 7.x
-@elastic/elasticsearch@7
 
-# Elasticsearch 6.x
-@elastic/elasticsearch@6
+==== Browser
+
+There is no official support for the browser environment since expose your Elasticsearch instance to everyone could lead to security issues.
```
Comment (lcawl) [inline docs/introduction.asciidoc]:
```suggestion
WARNING: There is no official support for the browser environment. It exposes your Elasticsearch instance to everyone, which could lead to security issues.
```

File: docs/introduction.asciidoc [hunk 7]
```diff
@@ -18,18 +19,129 @@ npm install @elastic/elasticsearch
...
+We recommend you to write a lightweight proxy that uses this client instead.
```
Comment (lcawl) [inline docs/introduction.asciidoc]:
```suggestion
We recommend that you write a lightweight proxy that uses this client instead.
```

**Comment by delvedor** [comment]
Thank you a lot @lcawl and @philkra!
